src/main/resources/site/pages/rss/rss.js

```diff
@@ -131,7 +131,11 @@ exports.get = function(req) {
 			content.data.include = libs.util.data.forceArray(content.data.include);
 			var includeLength = content.data.include.length;
 			for (var i = 0; i < includeLength; i++) {
-				query += ' AND _path LIKE "' + contentRoot + content.data.include[i] + '/*"';
+				if(i > 0) {
+					query += ' OR _path LIKE "' + contentRoot + content.data.include[i] + '/*"';
+				} else {
+					query += ' AND _path LIKE "' + contentRoot + content.data.include[i] + '/*"';
+				}
```

Comment:
Nice catch on the AND-problem! Is it also present for the "exclude path" function?

Already merged but another way of doing this would be to set a variable for AND | OR and pass it into the query instead of repeating the entire query.

Reply:
I've [fixed it](https://github.com/enonic/app-rss/blob/a9fffd5ed5b8f3d27f59434b3289032c84a2cd8d/src/main/resources/site/pages/rss/rss.js#L134) after the merge to not repeat the query.